java/java.navigation/src/org/netbeans/modules/java/navigation/ElementScanningTask.java

```diff
@@ -358,7 +360,7 @@ private Description element2description(final Element e, final Element parent,
         } else if( e instanceof ExecutableElement ) {
             d.htmlHeader = createHtmlHeader(info,  (ExecutableElement)e, info.getElements().isDeprecated(e),d.isInherited, fqn, overridenFrom);
         } else if( e instanceof VariableElement ) {
-            if( !(e.getKind() == ElementKind.FIELD || e.getKind() == ElementKind.ENUM_CONSTANT) )
+            if( !(e.getKind() == ElementKind.FIELD || e.getKind() == ElementKind.ENUM_CONSTANT || e.getKind().name().equals("STATE_COMPONENT")) )
```

Comment:
@singh-akhilesh should this be `RECORD_COMPONENT`? ElementKind does not have a `STATE_COMPONENT`.

Reply:
Hi @mbien STATE_COMPONENT was part of the amber project and  replace by RECORD_COMPONENT in the java14 https://hg.openjdk.java.net/amber/amber/rev/2a5e841bd91b. Will take necessary action.

Reply:
https://github.com/apache/netbeans/pull/4293